.circleci/config.yml

```diff
@@ -380,7 +380,7 @@ jobs:
 
   build-package:
     machine:
-      image: ubuntu-2004:202104-01
```

Comment:
I suppose 20.04 is causing build issues in our env but package can be installed fine on it? @bogdandrutu

Reply:
probably, but too lazy to debug now :)

Reply:
😬 I guess it's OK assuming it was always broken on 20.04 in whatever way